yos/exist.go

```diff
@@ -0,0 +1,77 @@
+package yos
+
+import (
+	"errors"
+	"os"
+	"path/filepath"
+)
+
+var (
+	ErrIsDir        = errors.New("target is a directory")
+	ErrIsFile       = errors.New("target is a file")
+	ErrIsNotSymlink = errors.New("target is not a symbolic link")
+)
+
+// IsExist checks if the file, directory exists.
+// If the file is a symbolic link, it will attempt to follow the link and check if the source file exists.
+func IsExist(path string) bool {
+	_, err := os.Stat(path)
+	return err == nil
+}
+
+// IsExist checks if the file, directory doesn't exist.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [comment on exported function IsNotExist should be of the form "IsNotExist ..."](https://app.codacy.com/app/an9an63/gut/pullRequest?prid=4635828)